src/cache.rs

```diff
@@ -87,7 +87,8 @@ impl<'font> GpuCache<'font> {
             };
 
             *result = upload(rect, data, queue, cmd, img, buf).map(Some);
-        })
+        });
```

Comment:
Should propagate the error with
```suggestion
        })?;
```